test/models/post_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class PostTest < ActiveSupport::TestCase
+  test "should not save a post without a title" do
+    post = Post.new(:body => "Jesse Ventura", :summary => "Pro wrestler, statesman")
+    assert_not post.save, "Save the post without a title."
+  end
+
+  test "should not save an author without a first name" do
```

Comment:
This isn't a test of `Post` is it? Shouldn't this (and the next one) find a better home in your `Author` tests?